alacritty_terminal/src/term/search.rs

```diff
@@ -245,27 +287,27 @@ impl<T> Term<T> {
                 };
 
                 // Since we get the state from the DFA, it doesn't need to be checked.
```

Comment:
Perhaps update this comment since it doesn't need to mention being checked anymore?